src/main/java/seedu/duke/data/ModuleManager.java

```diff
@@ -4,12 +4,10 @@
 import seedu.duke.exception.DuplicateDataException;
 import seedu.duke.exception.ModuleNotProvidedException;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 
 public class ModuleManager {
-    private static ArrayList<Module> moduleList;
-    private static HashMap<String, String> modulesMap;
+    private static HashMap<String, Module> modulesMap = new HashMap<>(); // Main module list. Maps module code to module object.
```

Comment:
I think it should contain two module list, (or maps) 
One module map is for storing the loaded module map from the NUSMODS
The other one is for storing the modules added by user.

Reply:
Okay, I will add that in a later PR